.env.sample

```diff
@@ -48,6 +48,10 @@ AWS_REGION=xx-xxxx-x
 AWS_S3_UPLOAD_BUCKET_URL=http://s3:4569
 AWS_S3_UPLOAD_BUCKET_NAME=bucket_name_here
 AWS_S3_UPLOAD_MAX_SIZE=26214400
+# upload images to s3 with private access controls
+# and serve them via signed urls
+# false by default
+ENABLE_PRIVATE_CONTENT=false
```

Comment:
I think private should be the default going forward – so you'd have to explicitly disable it, the only reason I can think to do so would be if you're using an S3 alternative that doesn't support the signing?

With that in mind perhaps this could become `AWS_S3_ACL=private` /  `AWS_S3_ACL=public-read` – i'd like to keep it related in the AWS_S3 namespace, if not set the app would assume private.